generators/ruby/faraday/templates/README.mustache

```diff
@@ -0,0 +1,104 @@
+# Onfido
+
+The official Ruby library for integrating with the Onfido API.
+
+[![Gem Version](https://badge.fury.io/rb/onfido.svg)](http://badge.fury.io/rb/onfido)
+[![Build Status](https://travis-ci.org/onfido/onfido-ruby.svg?branch=master)](https://travis-ci.org/onfido/onfido-ruby)
+
+Documentation can be found at https://documentation.onfido.com
+
+This version uses Onfido API v3.6 and is compatible with Ruby 2.4 onwards. Refer to our [API versioning guide](https://developers.onfido.com/guide/api-versioning-policy#client-libraries) for details of which client library versions use which versions of the API.
+
+## Installation & Usage
+
+### Requirements
+
+Ruby {{{generatorLanguageVersion}}}
+
+### Installation
+
+```ruby
+gem {{{gemName}}}, '~> 2.0.1'
+```
+
+Configure with your API token and region:
+
+```ruby
+require {{{gemName}}}
+
+Onfido.configure do |config|
+  config.api_token = ENV["ONFIDO_API_TOKEN"]
+  config.region = config.region[:EU]
+end
+
+onfido_api = Onfido::DefaultApi.new
+```
+
+All resources share the same interface when making API calls. Use `.create` to create a resource, `.find` to find one, and `.all` to fetch all resources.
+
+### Making a call to the API
+
+```ruby
+applicant = onfido_api.create_applicant(
+  first_name: 'Test',
+  last_name: 'Applicant'
+)
+```
+
+Documentation and code examples can be found at https://documentation.onfido.com
+
+## Error Handling
```

Comment:
Not sure about the error handling paragraph, i didn't update this one.

Reply:
Looking at [this](https://github.com/OpenAPITools/openapi-generator/blob/master/modules/openapi-generator/src/main/resources/ruby-client/api_client_faraday_partial.mustache#L29-L32) code, it seems [Faraday exceptions](https://www.rubydoc.info/github/lostisland/faraday/Faraday/ClientError) should be raised.